pkg/service/auth.go

```diff
@@ -26,7 +30,7 @@ func (self *AuthService) Check(ctx context.Context, req *auth.CheckRequest) (*au
 	if err != nil {
 		return self.deniedResponse(rpc.FAILED_PRECONDITION, "Invalid request"), nil
 	}
-	log.Println("[AuthService] Check()\n", string(reqJSON))
+	authServiceLog.Info("Check()", "reqJSON", string(reqJSON))
```

Comment:
IDK if works, but could we go straight to the value of request as JSON and skip the "reqJSON" tag?

```suggestion
	authServiceLog.Info("Check()", string(reqJSON))
```

Reply:
Sadly nope, the func signature expects a `msg` of type `string` then a key/value pairs as context.

> The msg argument should be used to add some constant description to the log line. The key/value pairs can then be used to add additional variable information. The key/value pairs should alternate string keys and arbitrary values